### PR TITLE
Minor collab link fix

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,11 +231,11 @@ model = models.get("model-name", pretrained_weights="pretrained-model-name")
 
 ### How to Predict Using Pre-trained Model
 
-* [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Deci-AI/super-gradients/blob/master/notebooks/how_to_use_knowledge_distillation_for_classification.ipynb) [Knowledge Distillation Training](https://github.com/Deci-AI/super-gradients/blob/master/notebooks/how_to_run_model_predict.ipynb) [How to Predict Using Pre-trained Model](https://github.com/Deci-AI/super-gradients/blob/master/notebooks/how_to_run_model_predict.ipynb)
+* [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Deci-AI/super-gradients/blob/master/notebooks/how_to_run_model_predict.ipynb) [How to Predict Using Pre-trained Model](https://colab.research.google.com/github/Deci-AI/super-gradients/blob/master/notebooks/how_to_run_model_predict.ipynb)
```
Comment (shaydeci) [inline README.md:234]: I think the point was to have 2 links - one for the colab and one for the direct link inside SG.

**Review by shaydeci — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Louis-Dupont) [reply README.md:234]: Don't we always only link to collab? 

Also, here there were 2 links, but one of them was completely off (see screenshot, the last links)

image

Basically we had 2 link, with 2 names. One of the two was not related to the section, so I removed it

**Review by shaydeci — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Louis-Dupont) [reply README.md]: So you say that the collab image should link to collab, but the text link to the github recipe, right ?
If so, I think Eugene and I always linked both to the collab, I personally thought this was the convention.

**Pull request merged**